lib/prometheus_exporter/client.rb

```diff
@@ -13,15 +13,28 @@ def initialize(name:, help:, type:, client:)
       @type = type
     end
 
-    def observe(value = 1, keys = nil)
-      @client.send_json(
+    def standard_values(value, keys)
+      {
         type: @type,
         help: @help,
         name: @name,
         keys: keys,
         value: value
-      )
+      }
     end
+
+    def observe(value = 1, keys = nil)
+      @client.send_json(standard_values(value, keys))
+    end
+
+    def increment(value = 1, keys = nil)
+      @client.send_json(standard_values(value, keys).merge(action: :increment))
```

Comment:
to avoid the "merge" and throwaway object I would just have standard values accept the action optionally, if nothing specified it should not add an action